src/test/resources/pages/instructorCourseEnrollError.html

```diff
@@ -60,7 +60,11 @@
                     </span>
                     <br>
                     <span class="problemDetail">
-                      • "bjack.gmail.tmt" is not acceptable to TEAMMATES as an email because it is not in the correct format. An email address contains some text followed by one '@' sign followed by some more text. It cannot be longer than 254 characters. It cannot be empty and it cannot have spaces.
+                      •
+                      <b>
+                        Email
+                      </b>
+                      must contain some text followed by one '@' sign followed by some more text. It cannot be longer than 254 characters and cannot have spaces.
```

Comment:
@damithc 
The details is actually in the earlier part of the error message:
![image](https://cloud.githubusercontent.com/assets/14101781/14059792/5a3f836a-f389-11e5-871c-cc3f08a6d5b5.png)

Do you think it's sufficient?